code/game/gamemodes/objective_items.dm

```diff
@@ -166,6 +166,15 @@
 		return 1
 	return 0
 
+/datum/objective_item/steal/specific
+	difficulty = 5
+
+/datum/objective_item/steal/specific/New()
+	..()
+	var/obj/target = pick(subtypesof(/obj/item/slime_extract))
+	name = initial(target.name)
+	targetitem = "an unused sample of [target]."

```

Comment:
shouldn't name be "an unused sample of [target]." and targetitem be target?
every other objective item has targetitem as an obj path

Reply:
probably yeah I wrote that at 2 am

Reply:
Needs fixing, if you're still on with this idea.